eng/Versions.props

```diff
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
     
-    <VersionPrefix>2.9.1</VersionPrefix>
+    <VersionPrefix>3.0.0</VersionPrefix>
```

Comment:
Our analyzer package version matches the version of referenced Microsoft.CodeAnalysis NuGet package. So this change should be accompanied by bumping up the CodeAnalysis version to 3.0.x